tools/travis/build.sh

```diff
@@ -21,19 +21,20 @@ $ANSIBLE_CMD apigateway.yml
 
 cd $ROOTDIR
 
-./gradlew distDocker -x :core:swift3Action:distDocker -x :core:pythonAction:distDocker -PdockerImagePrefix=testing
+time ./gradlew distDocker -PdockerImagePrefix=testing
 
 cd $ROOTDIR/ansible
 
 $ANSIBLE_CMD wipe.yml
+time docker commit couchdb "openwhisk/couchdb"
```

Comment:
should we call the commit `couchdb-snapshot` instead (to not confuse it with the images pushed to docker hub)?